### ISSUE TITLE
add PFR number from uniprot in display

### ISSUE THREAD

**Issue by trishorts** [smith-chem-wisc/proteoform-suite#420]
These are the proteoform records that Neil worked so hard to get UniProt to add. Would be very nice for us to link nodes to those things.

**Comment by lschaffer2** [comment]
Yes, this would help a lot with matching up with topdown id's, which each have the PFR #.

**Comment by acesnik** [comment]
Sounds like this number isn't referenced in UniProt anymore. We could put this on the "future" list, and query these accessions once we link with the CTDP database.

**Comment by trishorts** [comment]
Go to UniProt website and view the entry for Q03880

There is a block called "Proteomic databases"

In that block is a field TopDownProteomics1 with a hyperlink "Q03880"

Selecting that hyperlink takes you to CTDB website and shows all the PFR numbers for that protein.

**Comment by lschaffer2** [comment]
PFR numbers are based on sequence and PTM localization.... we could have a List<string> called "Potential PFR numbers" for identified intact-mass experimentals
